package.json

```
@@ -50,9 +50,9 @@
   },
   "engines": {
     "node": ">=14.17.6",
-    "pnpm": ">=8.7.0 <9"
+    "pnpm": ">=8.7.0"
```

How about limit to `>=9.0.0`

```suggestion
    "pnpm": ">=9.0.0"
```